.github/pull_request_template.md

```diff
@@ -1,14 +1,60 @@
+<!--  Thanks for sending a pull request!  Here are some tips for you:
+
+1. If this is your first time, please read our contributor guidelines: (TODO link CONTRIBUTING.md) and developer guide (TODO link dev guide)
+2. Make sure the to check these marks:
+  * [ ] Please, make sure to read CONTRIBUTING.md and CODE_OF_CONDUCT.md
+  * [ ] Make sure to open your PR against the right branch: master / release-VERSION
+  * [ ] Add to CHANGELOG.md relevant changes and any other user facing change.
+  * [ ] Make sure to sign-off all your commits
+  * [ ] GPG signing is appreciated
+  * [ ] Make sure the code follows coding style (use automated formatting, such as `mix format`)
+
+3. Please label this pull request according to what type of issue you are addressing, especially if this is a release targeted pull request.
+4. Ensure you have added or ran the appropriate tests for your PR
+5. If the PR is unfinished, mark it as `[WIP]` in the title
+-->
+
+#### What type of PR is this?
+
 <!--
+Add one of the following kinds:
+- fix
+- cleanup
+- documentation
+- feature
+-->
 
-**Please, carefully describe what the PR does and why you are opening it.**
+#### What this PR does / why we need it:
 
-Short check list:
+#### Which issue(s) this PR fixes:
+<!--
+*Automatically closes linked issue when PR is merged.
+Usage: `Fixes #<issue number>`, or `Fixes (paste link of issue)`.
+_If PR is about `failing-tests or flakes`, please post the related issues/tests in a comment and do not use `Fixes`_*
+-->
+Fixes #
 
-* [ ] Please, make sure to read CONTRIBUTING.md and CODE_OF_CONDUCT.md
-* [ ] Make sure to open your PR against the right branch: master / release-VERSION
-* [ ] Add to CHANGELOG.md relevant changes and any other user facing change.
-* [ ] Make sure to sign-off all your commits
-* [ ] GPG signing is appreciated
-* [ ] Make sure the code follows coding style (use automated formatting, such as `mix format`)
+#### Special notes for your reviewer:
 
+#### Does this PR introduce a user-facing change?
```

Comment:
should the user put the release note both in the pr description and the changelog?